WORKSPACE

```diff
@@ -16,13 +16,19 @@ workspace(name = "com_google_gapic_generator_cpp")
 
 load(
     "//:repositories.bzl",
+    "com_google_gapic_generator_cpp_gax_repositories",
     "com_google_gapic_generator_cpp_repositories",
 )
 
+com_google_gapic_generator_cpp_gax_repositories()
 com_google_gapic_generator_cpp_repositories()
 
 bind(
     name = "zlib",
     actual = "@net_zlib//:zlib",
 )
 
+# NOTE: this is a hack to allow grpc to build
```

Comment:
Does this go away once https://github.com/googleapis/gapic-generator-cpp/pull/31 is merged?

Reply:
Wait a minute, I'll go check.

Reply:
Yes. It's a known limitation of bazel.

Reply:
Wait, yes it goes away? Or yes it is still needed?

Reply:
Aaah! Bad phrasing/hasty reading on my part. Yes, it's still needed; no, it does not go away.